java/vector/pom.xml

```diff
@@ -135,11 +146,6 @@
             <goals>
               <goal>test</goal>
             </goals>
-            <configuration>
-              <classpathDependencyExcludes>
-                <classpathDependencyExclude>org.apache.arrow:arrow-memory-netty</classpathDependencyExclude>
-              </classpathDependencyExcludes>
-            </configuration>
```

Comment:
?

Reply:
I think since we are shading netty in the memory-netty module, we should not remove it from the classpath

Reply:
try to add it back and see if you get any error while executing tests

Reply:
I tried it and it fails the vector package with `siren.io.netty` not found.

Reply:
@scampi the `org.apache.arrow:arrow-memory-netty` needs to be excluded from the classpath as it is causing `jar hell `on siren-platform (after upgrading arrow to 4.0.0-1). The config above does not work and also trying `<classpathDependencyScopeExclude>test</classpathDependencyScopeExclude>` fails with `NoClassDefFoundError`

Reply:
The `jarhell` has been resolved by excluding the `memory-core` from the shaded netty in `memory-netty`, see commit https://github.com/sirensolutions/arrow/pull/16/commits/5ed587b3ad8d621238d5301be5ffa0f91a97cd8f

Reply:
Can this be added back ? Does the commit fixing the jarhell means this can be left as it was originally ?

Reply:
This configuration can't be added back, it causes `NoClassDefFoundError`